lib/lwt/lwt_file.mli

```diff
@@ -20,7 +20,7 @@
 
 open Js_of_ocaml
 open Js
-open File
```

Comment:
@trefis, this looks like a bug in OCaml 4.10
```
File "lib/lwt/lwt_file.mli", line 23, characters 0-9:
23 | open File
     ^^^^^^^^^
Error (warning 33): unused open Js_of_ocaml.File.
```

Reply:
Ack. I'll take a look when I get back to the office (i.e. Thursday).